app/policies/organisation_policy.rb

```diff
@@ -1,22 +1,22 @@
 class OrganisationPolicy < ApplicationPolicy
   def index?
-    user.administrator? || user.fund_manager?
+    user.administrator?
   end
 
   def show?
-    user.administrator? || user.fund_manager? || associated_user?
+    user.administrator? || associated_user?
```

Comment:
Do we need to keep the `associated_user?` check, as theoretically all users are admins and therefore associated users now. I've tried removing it and the tests still pass.

Reply:
You're right this doesn't make sense anymore. In the next pull request I switch the first part back to being a `beis_user?` so hopefully all is right again https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/199/files#diff-172620c2e4ee916f15c9384c3b1257abR7